exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/Foreman.java

```diff
@@ -149,7 +148,7 @@
   private final boolean queuingEnabled;
 
 
-  private String queryText;
+  private String queryText = "";
```

Comment:
May be it's better to check queryText nullability in QueryManager.java then setting empty string as default?